src/org/ods/services/NexusService.groovy

```diff
@@ -81,8 +81,15 @@ class NexusService {
 
     @NonCPS
     URI storeArtifact(String repository, String directory, String name, Path artifact, String contentType) {
-        InputStream inputStream = Files.newInputStream(artifact)
-        return storeArtifact(repository, directory, name, inputStream, contentType)
+        URI artifactURL
```

Comment:
Yo usaria el try-with-resources o bien haria unos tests...
Mas q nada pq al close se le suele poner un try and catch alrededor para q no pete y se suele mirar a ver si la variable tiene un valor != null...